packages/authgear-react-native/index.d.ts

```diff
@@ -995,31 +1038,67 @@ export declare class ReactNativeContainer {
    *
    * @param code - WeChat Authorization code.
    * @param state - WeChat Authorization state.
+   *
+   * @public
    */
   wechatAuthCallback(code: string, state: string): Promise<void>;
   /**
    * Check whether biometric is supported on the current device.
    * If biometric is not supported, then a platform specific error is thrown.
+   *
+   * @public
    */
   checkBiometricSupported(options: BiometricOptions): Promise<void>;
   /**
    * Check whether biometric was enabled for the last logged in user.
+   *
+   * @public
    */
   isBiometricEnabled(): Promise<boolean>;
+  /**
+   * Disable biometric authentication for the last logged in user.
+   * After disabling, the user may not be able to authenticate with biometric until it is enabled again.
+   *
+   * @public
+   */
   disableBiometric(): Promise<void>;
+  /**
+   * Enable biometric authentication for the last logged in user.
+   * Platform specific biometric authenticator will be prompted and it is enabled only if the verification is successful.
+   * You may first enable biometric authentication in Authgear Portal (Your Project \> Authentication \> Biometric \> Enable biometric authentication).
+   *

```

Comment:
You should also mention what error can be thrown.

Reply:
ok!